Rename tags to topics (#1575)
plone/documentation: docs/contributing/github-administration.md

`tooling`
: tools and applications needed to orchestrate Plone

## Manage topics

> To update the topics of a GitHub repository, go the the repository, i.e https://github.com/plone/plone.batching and click on the cog wheel on the `About` section on the right side of the screen.

Reviewer: ```suggestion
To update the topics of a GitHub repository, go to the repository's URL, such as https://github.com/plone/plone.batching, and click on the {guilabel}`gear icon` on the {guilabel}`About` section on the right side of the screen.
```
